featurebyte/query_graph/transform/offline_store_ingest.py

```diff
@@ -251,13 +197,13 @@ def _prepare_offline_store_ingest_query_specific_node_parameters(
         node_name_to_subgraph_node_name: Dict[str, str],
         aggregation_node_names: Set[str],
         aggregation_info: AggregationInfo,
-        entity_id_to_serving_name: Dict[PydanticObjectId, str],
-        catalog_id: PydanticObjectId,
+        operation_structure: OperationStructure,
     ) -> Dict[str, Any]:
         agg_nodes_info = []
         feature_job_settings = []
+        agg_node_names = [agg.node_name for agg in operation_structure.iterate_aggregations()]
         for node_name in aggregation_node_names:
-            if node_name in node_name_to_subgraph_node_name:
+            if node_name in agg_node_names:
```

Comment:
Update the condition here so that we use only those aggregation node specified in operation structure.